hoot-js/src/main/cpp/hoot/js/JavaScriptTranslator.cpp

```diff
@@ -160,7 +160,7 @@ vector<JavaScriptTranslator::TranslatedFeature> JavaScriptTranslator::_createAll
   QVariantList list)
 {
   vector<TranslatedFeature> result;
-  result.reserve(list.size());
+  //result.reserve(list.size());
```

Comment:
Remnant of a effort months ago to get code to work on CentOS.  Yum package for Qt4 was older than 4.8, so it didn't compile there.